api/graphite.go

```diff
@@ -568,7 +568,27 @@ func (s *Server) executePlan(ctx context.Context, orgId int, plan expr.Plan) ([]
 	// e.g. target=movingAvg(foo.*, "1h")&target=foo.*
 	// note that in this case we fetch foo.* twice. can be optimized later
 	for _, r := range plan.Reqs {
-		series, err := s.findSeries(ctx, orgId, []string{r.Query}, int64(r.From))
+		var err error
+		var series []Series
+		if strings.HasPrefix(r.Query, "seriesByTag(") {
+			startPos := strings.Index(r.Query, "(")
```

Comment:
at this point we know that `r.Query` starts with `seriesByTag(`, so we shouldn't need to scan the string for `(` because we already know the position